src/pages/o/[orgId]/contact.tsx

```diff
@@ -1,22 +1,21 @@
-//TODO: Enable eslint rule and fix errors
-/* eslint-disable  @typescript-eslint/no-non-null-assertion */
 import { dehydrate } from 'react-query/hydration';
 import { GetServerSideProps } from 'next';
 import { QueryClient } from 'react-query';
 
 import getOrg from '../../../fetching/getOrg';
-import { LayoutParams } from '../../../interfaces/LayoutParams';
 import OrgLayout from '../../../components/layout/OrgLayout';
+import { PageWithLayout } from '../../../types/index';
```

Comment:
The nice thing about naming a module `index` is that you don't have to include the name of the file when you import. Referencing the name of the directory it's in is enough. This line should just be:

```typescript
import { PageWithLayout } from '../../../types';
```

Reply:
Oh yes, forgot about this! Fixed.